.github/workflows/release.yml

```
@@ -0,0 +1,62 @@
+name: releaseing
+
+on:
+  push:
+    branches:
+      - release/major
+      - release/minor
+      - release/patch
+      - release/premajor
+      - release/preminor
+      - release/prepatch
+      - release/prerelease
+
+jobs:
+  releaseing:
+    name: releaseing
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```
IMO. v3を使っても良さそうでしょうか👀

このorgでも既に使われていそうでしたhttps://github.com/openameba/spindle.ameba.design/blob/88b05874578bceb4081ad72e7bb2a0b6f418fe5d/.github/workflows/test.yml#L12

---

https://github.com/openameba/cwv-logger/pull/1/commits/6328d5be59b77a93e6073511823f33fa21d5ae34
CI周りはこちらでまとめて修正しました！